docs/reference/mapping/runtime.asciidoc

```diff
@@ -199,18 +199,18 @@ to `boolean`, or if the runtime field is removed.
 === Define runtime fields in a search request
 You can specify a `runtime_mappings` section in a search request to create
 runtime fields that exist only as part of the query. You specify a script
-as part of the `runtime_mappings` section, just as you would if adding a
-runtime field to the mappings.
+as part of the `runtime_mappings` section, just as you would if
+<<runtime-mapping-fields,adding a runtime field to the mappings>>.
 
-Fields defined in the search request take precedence over fields defined with
-the same name in the index mappings. This flexibility allows you to shadow
-existing fields and calculate a different value in the search request, without
-modifying the field itself. If you made a mistake in your index mapping, you
-can use runtime fields to calculate values that override values in the mapping
-during the search request.
+Defining a runtime field in a search request uses the same format as defining
+a runtime field in the index mapping. That consistency means you can promote a
+runtime field from a search request to the index mapping by moving the field
+definition from `runtime_mappings` in the search request to the `runtime`
+section of the index mapping.
```

Comment:
I think it'd be less "product-y" to say something like "Defining a runtime field in a search request uses the same format as defining a runtime field in the index mapping. So you can copy and paste from one to the other."  Or something like that. I dunno. That's partly me putting my casual voice into it. Maybe its just me, but I think its kind of obvious that we did this intentionally and we don't need to sounds pedantic about it? I dunno.

Reply:
That's a good callout. How about something like:
>Defining a runtime field in a search request uses the same format as defining a runtime field in the index mapping.
Just copy the field definition from the `runtime_mappings` in the search request to the `runtime` section of the index mapping.

Reply:
Sounds good to me!